Algebras/DotBracket/Parts/algpart_cofold_dotBracket_basic.gap

```diff
@@ -0,0 +1,137 @@
+  string sadd(Subsequence lb,string e) {
```

Comment:
what is the difference between this file and `algpart_dotBracket_basic.gap`? And why do we need both?

Reply:
The difference is, that in `algpart_dotBracket_basic.gap` there is 
a) the addss-function which I don't need in my new grammar and
b) the ml-function, whose inputs I changed in my new grammar
That's why I created a new file where I deleted those two functions.

Is there a better way to do this without disturbing the nodangle.gap?